PWGGA/GammaConvBase/AliConversionMesonCuts.h

```diff
@@ -345,7 +346,7 @@ class AliConversionMesonCuts : public AliAnalysisCuts {
   private:
 
     /// \cond CLASSIMP
-    ClassDef(AliConversionMesonCuts,45)
+    ClassDef(AliConversionMesonCuts,46)
```

Comment:
not needed, keep 45

Reply:
Then I think I don't fully understand it. I added a new function, I thought that is a reason for an increment.

Reply:
increment is only needed when adding/removing persistent member variables

Reply:
But what does persistent mean in that case? 
Also, the PR seems bugged for me, since there is no merging after the +1

Reply:
Basically every member variable which does not have a //! (meaning it is transient)
Yes, there were some general issues with PRs - should work soon